### DISCUSSION TITLE
How use import ?

### DISCUSSION THREAD

**Discussion by Jengro777** [vlang/v#21153] [General]
```
import os
import strings
import mymodule.submodule 
```

Can I import it in the following way
`import /src/meili/search.v`

**Comment by JalonSolov** [comment]
No.

**Reply by Jengro777** [reply]
![image](https://github.com/vlang/v/assets/35259040/6a2ffb0a-f355-4c12-87ad-8e4c6c8b60a0)
![image](https://github.com/vlang/v/assets/35259040/f28b9b6f-0fb4-4723-9961-e12a2745bb52)
But I have been following the tutorial and cannot reference functions in the module

**Reply by JalonSolov** [reply]
Do you have a `v.mod` file at the root of the project (as in, in the parent of the `src` dir)?  That is the anchor V uses to find local modules.

**Reply by Jengro777** [reply]
![image](https://github.com/vlang/v/assets/35259040/e12cc471-566d-438b-86ea-5f26863dcfa0)

**Reply by JalonSolov** [reply]
That appears to be inside the `src` dir.  Try moving your `v.mod` file to the parent of the `src` dir.

**Reply by Jengro777** [reply]
![image](https://github.com/vlang/v/assets/35259040/63e70a96-a3f8-4f0c-a50d-8e0e4f2452d0)

v. The mod should be in the root directory of the project and there should be no problem. The default location when I use 'v new vpod' to create a project

**Reply by Jengro777** [reply]
`If the code of your project is in multiple files, try with `v .` instead of `v .\main_mall.v``

I use v run \Main_mall. v runs, but I don't know if running multiple files should use v run

Now it's ready. Thank you very much for your answer

**Reply by JalonSolov** [reply]
Using `v <specific V file>` will only include that V file and anything it imports.  Using `v <directory name>` (where `.` means current directory) will compile _all_ V files in the directory, plus all imports.
